scripts/provision-member-directories.sh

```diff
@@ -145,16 +145,17 @@ echo "Writing codeowners file"
 * @ministryofjustice/modernisation-platform
 EOL
 
+
   for file in $environment_json_dir/*.json; do
     application_name=$(basename "$file" .json)
     directory=/terraform/environments/$application_name
     account_type=$(jq -r '."account-type"' ${environment_json_dir}/${application_name}.json)
-    codeowners=$(jq -r 'try (.codeowners[] | "@ministryofjustice/" + .)' ${environment_json_dir}/${application_name}.json | sort | uniq | tr '\n' ' ')
+    codeowners=$(jq -r 'try (.codeowners[] | select(length > 0) | "@ministryofjustice/" + .)' ${environment_json_dir}/${application_name}.json | sort | uniq | tr '\n' ' ')
     github_slugs=$(jq -r '.environments[].access[].github_slug | "@ministryofjustice/" + .' ${environment_json_dir}/${application_name}.json | sort | uniq | tr '\n' ' ')
 
     if [ "$account_type" = "member" ]; then
       # if codeowners array has been defined in the json file, use that
-      if [ -n "$codeowners" ]; then
+      if [ -n "$codeowners" ] && ["$missing_codeowners" -ne 0 ]; then
```

Comment:
Is this needed?  I can't see where it is defined

Reply:
Bum. I thought I'd tidied that up too. It'll be up in a few minutes

Reply:
New one posted